libs/features/profile/src/lib/features-profile.scss

```diff
@@ -8,4 +8,7 @@
     margin-top: 5px;
     margin-bottom: 10px;
   }
+  .MuiDialogContent-root:first-child {
+    padding-top: 0 !important;
```

Comment:
I would prefer to avoid !important if possible. Write full class/elements path instead.